spec/mysql2/error_spec.rb

```diff
@@ -15,6 +15,8 @@
     error
   end
 
+  after(:each) { client.close }
+
```

Comment:
I didn't know that.  All the other test files use `after(:each)`, so should I keep it as `after(:each)` for consistency?